functions-methods.js

```diff
@@ -28,9 +50,23 @@
 // * Er een @ in voorkomt
 // * Er géén , in voorkomt
 // * Er géén . in voorkomt als allerlaatste karakter (dus hotmail.com is valide, net als outlook.nl, maar outlooknl. niet)
+
+function checkEmailValidity(emailadres)
+{
+    if (emailadres.includes("@") && emailadres.includes(".") && !emailadres.endsWith("."))
+    {
+        console.log(true);
+    }else if (emailadres.includes(",") || !emailadres.includes("@") || emailadres.endsWith("."))
+    {
+        console.log(false);
+    }
```

Comment:
Je zou dit nog wat korter kunnen schrijven
```suggestion
    return emailadres.includes("@") && emailadres.includes(".") && !emailadres.endsWith(".") && !emailadres.includes(",");
```
De aanroeper kan het dan vervolgens in de console wegschrijven